Sources/PinLayout.swift

```diff
@@ -16,7 +16,7 @@
 //  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 //  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //  THE SOFTWARE.
-
+/*
```

Comment:
Temporarily commented out, should be removed

Reply:
Keep the file for now, we could do that in another PR